modules/core/src/main/java/org/apache/ignite/internal/processors/cluster/GridClusterStateProcessor.java

```diff
@@ -1617,6 +1643,7 @@ public ExchangeActions autoAdjustExchangeActions(ExchangeActions exchActs) {
                 ctx.localNodeId(),
                 null,
                 ClusterState.active(clusterState.state()) ? clusterState.state() : ACTIVE,
+                true,
```

Comment:
It's not clear why we force deactivation here.

Reply:
> It's not clear why we force deactivation here.

All previous internal changes of cluster state were forced. Thus, all current internal changes of cluster are considered forced. Only the exceptions are the user commands being translated to not-forced internal calls without flag ‘force’ passed.